pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -319,6 +319,10 @@ in
     dontUseCmakeConfigure = true;
   };
 
+  sassc = attrs: {
+    nativeBuildInputs = [ rake ];
```

Comment:
could it be an issue if rake was running a different version of ruby?

Reply:
I don't think so – IIRC rake is only used to compile libsass.